pytest_django/plugin.py

```diff
@@ -66,6 +67,9 @@ def pytest_addoption(parser):
     group._addoption('--migrations',
                      action='store_false', dest='nomigrations', default=False,
                      help='Enable Django migrations on test setup')
+    group._addoption('--django-debug-mode',
+                     action='store', dest='djangodebugmode', default='False',
+                     help='Configure the DEBUG setting. Defaults to False')
```

Comment:
Shouldn't it default to `None`?
Also punctuation at the end?!

Reply:
Done.